<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tp_packageProfile</class>
 <widget class="QWidget" name="tp_packageProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>206</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MiSans Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>MiSans Medium</family>
            </font>
           </property>
           <property name="text">
            <string>包名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>MiSans Medium</family>
            </font>
           </property>
           <property name="text">
            <string>关联文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>MiSans Medium</family>
            </font>
           </property>
           <property name="text">
            <string>安装者</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_name">
           <property name="font">
            <font>
             <family>MiSans Medium</family>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_path">
           <property name="font">
            <font>
             <family>MiSans Medium</family>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_installer">
           <property name="font">
            <font>
             <family>MiSans Medium</family>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="uninstall">
         <property name="font">
          <font>
           <family>MiSans Medium</family>
          </font>
         </property>
         <property name="text">
          <string>卸载</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear">
         <property name="font">
          <font>
           <family>MiSans Medium</family>
          </font>
         </property>
         <property name="text">
          <string>清除所有数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="enable">
         <property name="font">
          <font>
           <family>MiSans Medium</family>
          </font>
         </property>
         <property name="text">
          <string>启用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disable">
         <property name="font">
          <font>
           <family>MiSans Medium</family>
          </font>
         </property>
         <property name="text">
          <string>停用</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
